src/main/java/mate/academy/springbootstore/model/Book.java

```diff
@@ -1,4 +1,43 @@
 package mate.academy.springbootstore.model;
 
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "books")
 public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    @Column(nullable = false)
+    private String title;
+
+    @NotBlank
+    @Column(nullable = false)
+    private String author;
+
+    @NotBlank
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    @NotNull
+    @Positive
+    @Column(nullable = false)
+    private Double price;

```

Comment:
```suggestion
    private BigDecimal price;
```
Use BigDecimal for the price